interfaces/acados_matlab_octave/setup_AcadosOcp_from_legacy_ocp_description.m

```diff
@@ -74,16 +74,16 @@
     ocp.solver_options.nlp_solver_ext_qp_res = opts_struct.nlp_solver_ext_qp_res;
     ocp.solver_options.nlp_solver_step_length = opts_struct.nlp_solver_step_length;
     ocp.solver_options.globalization = upper(opts_struct.globalization);
-    ocp.solver_options.alpha_min = opts_struct.alpha_min;
-    ocp.solver_options.alpha_reduction = opts_struct.alpha_reduction;
-    ocp.solver_options.line_search_use_sufficient_descent = opts_struct.line_search_use_sufficient_descent;
+    ocp.solver_options.globalization_alpha_min = opts_struct.globalization_alpha_min;
```

Comment:
The options in the legacy interface should not be changed.
I.e. the option names on the right hand side here.

Reply:
I changed it back